examples/.examples-ci.yml

```diff
@@ -0,0 +1,77 @@
+examples:
+  stage: examples
+  needs: []
+  parallel:
+    matrix:
+      - DOTNET_VERSION: ['6.0']
+        EXAMPLE:
+          - Audit/AuditLog
+          - Audit/AuditLogBulk
+          - Audit/AuditLogBulkAsync
+          - Audit/AuditLogCustomSchema
+          - Audit/AuditLogNsign
+          - Audit/AuditMulticonfig
+          - Audit/AuditRoot
+          - Audit/AuditSearch
+          - Audit/AuditSearchCustomSchema
+          - AuthN/InviteActions
+          - AuthN/UserActions
+          - Embargo/IPCheck
+          - Embargo/ISOCheck
+          - FileScan/Scan_Async
+          - FileScan/Scan_RequestUploadURL_Post
+          - FileScan/Scan_RequestUploadURL_Put
+          - FileScan/Scan_Sync
+          - Intel/Defang/Defang
+          - Intel/Domain/Reputation
+          - Intel/Domain/ReputationBulk
+          - Intel/File/Reputation
+          - Intel/File/ReputationBulk
+          - Intel/File/ReputationFilepath
+          - Intel/IP/Geolocate
+          - Intel/IP/GeolocateBulk
+          - Intel/IP/GetDomain
+          - Intel/IP/GetDomainBulk
+          - Intel/IP/IsProxy
+          - Intel/IP/IsProxyBulk
+          - Intel/IP/IsVPN
+          - Intel/IP/IsVPNBulk
+          - Intel/IP/Reputation
+          - Intel/IP/ReputationBulk
+          - Intel/URL/Reputation
+          - Intel/URL/ReputationBulk
+          - Intel/User/PasswordBreached
+          - Intel/User/UserBreachedByEmail
+          - Intel/User/UserBreachedByIP
+          - Intel/User/UserBreachedByPhone
+          - Intel/User/UserBreachedByUsername
+          - Redact/RedactMulticonfig
+          - Redact/Structured
+          - Redact/Text
+          - Vault/Encrypt
+          - Vault/EncryptStructured
+          - Vault/Rotate
+          - Vault/Sign
+  image: mcr.microsoft.com/dotnet/sdk:${DOTNET_VERSION}
+  before_script:
+    - export PANGEA_AUDIT_CONFIG_ID="$(eval echo \$PANGEA_AUDIT_CONFIG_ID_1_LVE)"
```

Comment:
export PANGEA_AUDIT_CONFIG_ID="${PANGEA_AUDIT_CONFIG_ID_1_LVE}" should work right?

Reply:
Right, done.